01-shapefile-attributes.Rmd

```diff
@@ -6,8 +6,11 @@ authors: [Joseph Stachelek, Leah Wasser]
 contributors: [Megan A. Jones, Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
+category: 
 tags: [module-1]
+mainTag: GIS-Spatial-Data
 description: "This post explains the nature of shapefile attributes. Participants will be able to locate and query shapefile attributes as well as subset shapefiles by specific attribute values."
```

Comment:
@mjones01 let's make sure all descriptions refer to the pages as "lessons" rather than posts.